samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -162,6 +162,17 @@ BT_CONN_CB_DEFINE(conn_callbacks) = {
 	.security_changed = security_changed,
 };
 
+static void pairing_complete(struct bt_conn *conn, bool bonded)
+{
+	if (bonded) {
```

Comment:
Should we do it also if we are in not-discoverable hide UI indication advertising?

Reply:
Probably not... Thanks